src/main/java/com/gl/donate_receive/donate_receive/model/User.java

```diff
@@ -0,0 +1,21 @@
+package com.gl.donate_receive.donate_receive.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
```

Comment:
That imports too many things that aren't needed in an Entity. Please refrain from using wildcards unless you have 7-10 classes from the same package and that package isn't a root package. EG: javax.servlet.*; or org.mockito.*